torchx/schedulers/utils/ray_driver.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import logging
+import subprocess
+from ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, command, env):
+        self.command = command
+        self.env = env
+
+    def run_command(self):
+        return subprocess.run(self.command, env = self.env)
+
+def load_actor_json(filename : str) -> List[Dict]:
+    with open(filename) as f:
+        actor = json.load(f)
+        actor = json.loads(actor)
```

Comment:
If we can use something like `dataclasses-json` as I mentioned earlier, we can return a `RayActor` here, instead of a `dict`. This is also why I suggested moving `RayActor` to `ray_common.py`, so that it can be shared by both `ray_scheduler.py` and `ray_driver.py`.